README.md

```diff
@@ -1,2 +1,25 @@
-# operator-courier
-Build, verify and push operators
+# Operator Courier
+
+The Operator Courier is used to build, validate and push Operator Artifacts.
+
+## Building the tool locally with pip
+```bash
+$ pip install --user .
+
+$ operator-courier
+Hello, world. The build feature is currently under construction.
+Hello, world. The push feature is currently under construction.
+Hello, world. The validate feature is currently under construction.
```

Comment:
This is just going to change as soon as any of this is implemented, right? Why not just leave the output out of the readme?

Reply:
Addressed in the latest commit.